pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeClusterIntegrationTest.java

```diff
@@ -154,6 +156,7 @@ public void testInstanceShutdown()
   public void tearDown()
       throws Exception {
     dropRealtimeTable(getTableName());
+    testTableDataManagerCleanup();
```

Comment:
Rename it to `cleanupTestTableDataManager`?

Reply:
Renamed